fiftyone/utils/huggingface.py

```diff
@@ -63,6 +64,26 @@
 
 logger = logging.getLogger(__name__)
 
+def list_hub_datasets():
+    """Lists all FiftyOne datasets available on the Hugging Face Hub.
+
+    This function prints instructions for loading datasets and displays a list of
+    all datasets published by Voxel51 on the Hugging Face Hub.
+    """
+    from huggingface_hub import list_datasets
```

Comment:
Use `hfh.list_datasets` here so that the user will get the informative error message that `lazy_import()` enables if they don't have the `huggingface_hub` package installed